apps/web-client/src/lib/stores/inventory/__tests__/table_content.test.ts

```diff
@@ -24,4 +29,187 @@ describe('tableContentStore', () => {
+			year: '2009',
+			price: 22.0
+		});
+		const note = await db.warehouse().note('note-1').create();
+
+		await note.addVolumes(
+			['0195399706', 12, `v1/jazz`],
+			['019976915X', 10, `v1/jazz`],
+			['0195071409', 5, `v1/jazz`]
+		);
+
+		const de$ = createDisplayEntriesStore(db, note, readable(0), {});
+		let displayEntries: DisplayRow[];
+		de$.subscribe((de) => (displayEntries = de));
+		const want = [
+			{
+				isbn: '0195071409',
+				quantity: 5,
+				warehouseId: `v1/jazz`,
+				warehouseName: 'not-found',
+				title: 'Saving Darwin',
+				authors: 'Karl Giberson',
+				publisher: 'Harper Collins',
+				year: '2009',
+				price: 22.0
+			},
+
+			{
+				isbn: '0195399706',
+				quantity: 12,
+				warehouseId: `v1/jazz`,
+				warehouseName: 'not-found',
+				title: 'The Age of Wonder',
+				authors: 'Richard Holmes',
+				publisher: 'HarperCollins UK',
+				year: '2008',
+				price: 69.99
+			},
+			{
+				isbn: '019976915X',
+				quantity: 10,
+				warehouseId: `v1/jazz`,
+				warehouseName: 'not-found',
+				title: 'Twelve Bar Blues',
+				authors: 'Patrick Neate',
+				publisher: 'Penguin UK',
+				year: '2002',
+				price: 39.86
+			}
+		];
+
+		await waitFor(() => expect(displayEntries).toEqual(want));
+	});
+
+	test('should create display store only from entries streams if corresponding books do not exist', async () => {
+		const db = await newTestDB();
+
+		await db.upsertBook({
+			isbn: '0195399706',
+			title: 'The Age of Wonder',
+			authors: 'Richard Holmes',
+			publisher: 'HarperCollins UK',
+			year: '2008',
+			price: 69.99
+		});
+
+		await db.upsertBook({
+			isbn: '0195071409',
+			title: 'Saving Darwin',
+			authors: 'Karl Giberson',
+			publisher: 'Harper Collins',
+			year: '2009',
+			price: 22.0
+		});
+		const note = await db.warehouse().note('note-1').create();
+
+		await note.addVolumes(
+			['0195399706', 12, `v1/jazz`],
+			['019976915X', 10, `v1/jazz`],
+			['0195071409', 5, `v1/jazz`]
+		);
+
+		const de$ = createDisplayEntriesStore(db, note, readable(0), {});
+		let displayEntries: DisplayRow[];
+		de$.subscribe((de) => (displayEntries = de));
+		const want = [
+			{
+				isbn: '0195071409',
+				quantity: 5,
+				warehouseId: `v1/jazz`,
+				warehouseName: 'not-found',
+				title: 'Saving Darwin',
+				authors: 'Karl Giberson',
+				publisher: 'Harper Collins',
+				year: '2009',
+				price: 22.0
+			},
+
+			{
+				isbn: '0195399706',
+				quantity: 12,
+				warehouseId: `v1/jazz`,
+				warehouseName: 'not-found',
+				title: 'The Age of Wonder',
+				authors: 'Richard Holmes',
+				publisher: 'HarperCollins UK',
+				year: '2008',
+				price: 69.99
+			},
+			{
+				isbn: '019976915X',
+				quantity: 10,
+				warehouseId: `v1/jazz`,
+				warehouseName: 'not-found'
+			}
+		];
+
+		await waitFor(() => expect(displayEntries).toEqual(want));
+	});
+
+	test('should return book if found and undefined otherwise', async () => {
+		const db = await newTestDB();
+
+		const book1 = {
```

Comment:
Let's remove this test: it should be in `db` package, but let's just remove it for now (I'll write it up for enhancement ticket)